apps/docs/pages/guides/auth/auth-password-reset.mdx

```diff
@@ -0,0 +1,107 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+import { Accordion } from 'ui'
+
+export const meta = {
+  id: 'auth-password-reset',
+  title: 'Password Reset',
+  description: 'How to reset your password for your Supabase project',
+}
+
+- Send email for password reset using [`.resetPasswordForEmail`](/docs/reference/javascript/auth-resetpasswordforemail) while providing a `redirectTo` parameter
+- Email link will work as a magic link and log the user in then take them to the url specified in the `redirectTo` parameter
+- Create form to update the password and call the [`.updateUser`](/docs/reference/javascript/auth-updateuser) method with the new password
+
+## Single Page Application (SPA)
+
+### Sending password reset email
+Supabase provides a convenient method [`.resetPasswordForEmail`](/docs/reference/javascript/auth-resetpasswordforemail) 
+to reset a user password. This method takes a parameter of `redirectTo` which we will use to pass an absolute URL to 
+the update password page. This url must be saved in your allowed [Redirect URLs](https://app.supabase.com/project/sgbluorgiylcyqpqihab/auth/url-configuration) 
```

Comment:
```suggestion
the update password page. This url must be saved in your allowed Redirect URLs 
```

Not sure if the user will be redirected to the page if they don't own it. Consider omitting or adding an image instead

Reply:
Fixed, I forgot to use the `_` in the URL so it will work for anyone.